wrappers/services.go

```diff
@@ -456,3 +458,88 @@ func renderListenStream(socket *snap.SocketInfo) string {
 	listenStream := strings.Replace(socket.ListenStream, "$SNAP_DATA", snap.DataDir(), -1)
 	return strings.Replace(listenStream, "$SNAP_COMMON", snap.CommonDataDir(), -1)
 }
+
+func makeAbbrevWeekdays(start time.Weekday, end time.Weekday) []string {
+	out := make([]string, 0, 7)
+	for w := start; w%7 != (end + 1); w++ {
+		out = append(out, time.Weekday(w % 7).String()[0:3])
+	}
+	return out
+}
+
+// generateOnCalendarSchedules converts a timer specification into OnCalendar
+// schedules suitable for use in systemd *.timer units
+func generateOnCalendarSchedules(timer string) ([]string, error) {
```

Comment:
Nitpick, have it take a schedule object as argument so that caller has to parse it. This will just make it easier to test and compose.

Reply:
This is still the same, not sure if you feel strongly about it or if it was lost in review.

Reply:
Heh, the comment got hidden by github's UI when I pushed the commit with link to systemd man page.